test/Core/Src/main.c

```diff
@@ -1222,22 +1280,45 @@ void LCD_Delay(uint32_t Delay)
 
 /* USER CODE BEGIN Header_TouchGFX_Task */
 
-uint8_t *jsonString;
-
+char *jsonString;
+char *isNutrimax;
+int nutrimaxJsonStart = 0;
+int nutrimaxJsonStop = 0;
 void Button_Task(void *argument)
 {
 
+
 	jsonString = create_monitor();
-	//supports_full_hd(jsonString);
- //*jsonString = create_monitor();
   for(;;)
   {
+	  if (esp32ConnectionInit == TRUE) {
```

Comment:
Ja bi rađe ovo stavio u neki drugi task, Button task beš nema neke veze s komunikacijom s CNom.